captchas/sources.txt

```diff
@@ -2,4 +2,7 @@ CodeProject CAPTCHA
 http://www.codeproject.com/Articles/5947/CAPTCHA-Image 
 
 CryptograPHP CAPTCHA
-[Website no longer exists]
\ No newline at end of file
+[Website no longer exists]
+
+freeCap
+http://www.puremango.co.uk
```

Comment:
I'd rather not include this link until their site is up and running. If they're ever back up, I'll add it then.